tests/gnrc_ipv6_ext_opt/main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2015-2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Tests extension header handling of gnrc stack.
```

Comment:
```suggestion
 * @brief       Tests IPv6 hop-by-hop and destination option extension header handling of gnrc stack.
```

Reply:
I'll leave the "hop-by-hop and destination". It should be clear without it what is meant ;-).